cpp/src/plasma/client.h

```diff
@@ -121,31 +122,40 @@ class ARROW_EXPORT PlasmaClient {
   ///        device_num = 1 corresponds to GPU0,
   ///        device_num = 2 corresponds to GPU1, etc.
   /// \return The return status.
-  Status Create(const ObjectID& object_id, int64_t data_size, uint8_t* metadata,
+  ///
+  /// The returned object must be released once it is done with, then it
+  /// must be either sealed or aborted.
```

Comment:
It's ok for it to be sealed before it is released.

Maybe "The returned object must be released once it is done with. It must also be either sealed or aborted."?

Reply:
Indeed, it seems you're right.